database/rrdfunctions.h

```diff
@@ -0,0 +1,19 @@
+#ifndef NETDATA_RRDFUNCTIONS_H
+#define NETDATA_RRDFUNCTIONS_H 1
+
+#include "rrd.h"
+
+extern void rrdfunctions_init(RRDHOST *host);
+extern void rrdfunctions_destroy(RRDHOST *host);
+
+extern void rrd_collector_started(void);
+extern void rrd_collector_finished(void);
+
+extern void rrd_collector_add_function(RRDSET *st, const char *name, const char *help, const char *format, int timeout, bool sync, int (*function)(BUFFER *wb, int timeout, const char *function, void *collector_data, void (*callback)(BUFFER *wb, int code, void *callback_data), void *callback_data), void *collector_data);
```

Comment:
@ktsaou it's worthwhile `typedef`-ing callbacks to make the code more concise and easier to skim through.